packages/integrations/vue/src/index.ts

```diff
@@ -47,6 +49,12 @@ function virtualAppEntrypoint(options?: Options): Plugin {
 		},
 		configResolved(config) {
 			root = config.root;
+			if (options?.appEntrypoint) {
+				appEntrypoint = options.appEntrypoint.startsWith('.')
+					? path.resolve(root, options.appEntrypoint).replace(root, '')
+					: options.appEntrypoint;
+				appEntrypoint = appEntrypoint?.replaceAll('\\', '/')
+			}
```

Comment:
Is there a reason the path resolving code is changed here? e.g. the `.replace(root, '')` part and the`replaceAll('\\', '/')` part

Reply:
Ah you're right, I do not need to do either of these.